### PR TITLE
fix: support updating stroke color for text by typing in color picker input

### PR DESCRIPTION
**Pull request by ad1992**
This has been an existing issue for text editor 
Before
Excalidraw (20)
Now
Excalidraw (21)

### CODE REVIEW

**Comment by dwelle** [comment]
Oh yes, I noticed that just yesterday as well! 👍

File: src/element/textWysiwyg.tsx [hunk 1]
```diff
@@ -488,18 +488,21 @@ export const textWysiwyg = ({
     // in that same tick.
     setTimeout(() => {
       editable.onblur = handleSubmit;
-      // case: clicking on the same property → no change → no update → no focus
-      editable.focus();
```
Comment (dwelle) [inline src/element/textWysiwyg.tsx:492]: this one still needs to be there. Does the fix not work while keeping this?

Comment (ad1992) [reply src/element/textWysiwyg.tsx:492]: No the editor gets focussed so typing gets reflected on editor instead, I am not sure why do we need this ?

Comment (dwelle) [reply src/element/textWysiwyg.tsx:492]: When you click on an already selected button (e.g. you double-click on the same font button, or you open & close the color picker) there's no update which would trigger programmatic focus:

https://github.com/excalidraw/excalidraw/blob/c2a8712593a30de5c5785c44a08e0fd1eb7f0fce/src/element/textWysiwyg.tsx#L518-L521

Don't recall why the `pointerdown` preventing on the island isn't enough.

**Comment by dwelle** [comment]
There's a strange behavior (likely relating to updates) where once you type a valid color into the HEX input, it blurs it 😄

**Comment by ad1992** [comment]
> There's a strange behavior (likely relating to updates) where once you type a valid color into the HEX input, it blurs it 😄

fixed

**Review by dwelle — APPROVED** [review]

**Pull request merged**